trymito.io/pages/use-cases/python-training.tsx

```diff
@@ -0,0 +1,320 @@
+                    <video autoPlay loop disablePictureInPicture playsInline webkit-playsinline="true" muted>
+                        <source src="/python-training/vlookup.mp4"/>
+                    </video>
+                    </div>
+                </section>
+
+                <section>
+                    <div className={classNames(pageStyles.subsection, pageStyles.left_aligned_section)}>
+                        <h2 className={classNames('margin-top-4rem', 'margin-bottom-2rem', pythonTrainingStyles.three_quarter_width_header)}>
+                            Increase your Python training retention by embracing your business analyst&apos;s Excel mastery.
+                        </h2>
+                    </div>
+                    <div className={pageStyles.subsection}>
+                        <div className={securityStyles.security_bullet_container}>
+                            <div className={securityStyles.icon}>
+                                <Image className={securityStyles.icon} src={fastClockIcon} alt='icon'></Image>
+                            </div>
+                            <h3>
+                                Write Python code 4x faster
+                            </h3>
+                            <p>
+                                Mito generates the equivalent Python code for every edit the user makes. Business analysts <a className={pageStyles.link} href='https://www.trymito.io/blog/quantifying-mitos-impact-on-analyst-python-productivity' target="_blank" rel="noreferrer">write Python code 400% faster using Mito</a> compared to writing code by hand. 
+                            </p>
+                            <p>
+                                The biggest blocker for analysts building their own automation after a training is not having enough time to do so. Mito solves that problem. Empower your analysts to write <code>=VLOOKUP</code> in Mito instead of struggling through <a className={pageStyles.link} href='https://www.trymito.io/excel-to-python/functions/lookup/VLOOKUP#Implementing%20VLOOKUP%20in%20Pandas' target="_blank" rel="noreferrer">5 lines</a> of complex pandas code. 
+                            </p>
+                        </div>
+                        <div className={classNames(securityStyles.security_bullet_container, pageStyles.subsection_second_element_mobile_spacing)}>
+                            <div className={securityStyles.icon}>
+                                <Image className={securityStyles.icon} src={automationIcon} alt='icon'></Image>
+                            </div>
+                            <h3>
+                                Reduce time to useful automation
+                            </h3>
+                            <p>
+                                The best way to convince business analysts to commit to Python training is to start saving them time as fast as possible. 
```

Comment:
This sentence is a little weird. Let's flip it. `Mito allows analysts to automate their real workflows from day one. This is the fastest way to demonstrate Python's value`. Or something like thsi